tests/test_serializers_benchmarks.py

```diff
@@ -0,0 +1,115 @@
+from decimal import Decimal
+
+from pytest import mark
+
+from datetime import datetime
+
+from rest_framework import serializers
+from .models import RegularFieldsModel, RegularFieldsAndFKModel
+
+
+data = {
+    'big_integer_field': 100000,
+    'char_field': 'a',
+    'comma_separated_integer_field': '1,2',
+    'date_field': datetime.now().date(),
+    'datetime_field': datetime.now(),
+    'decimal_field': Decimal('1.5'),
+    'email_field': 'somewhere@overtherainbow.com',
+    'float_field': 0.443,
+    'integer_field': 55,
+    'null_boolean_field': True,
+    'positive_integer_field': 1,
+    'positive_small_integer_field': 1,
+    'slug_field': 'slug-friendly-text',
+    'small_integer_field': 1,
+    'text_field': 'lorem ipsum',
+    'time_field': datetime.now().time(),
+    'url_field': 'https://overtherainbow.com'
+}
+
+
+class TestSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = RegularFieldsModel
+        fields = data.keys() + ['method']
```

Comment:
This doesn't work with Python3 since keys are an inferable. You can work around this by using `list(data.keys())` for Python 2/3